packages/trader/src/App/Components/Elements/PositionsDrawer/PositionsDrawerCard/positions-drawer-card.jsx

```diff
@@ -113,6 +113,44 @@ const PositionsDrawerCard = ({
         </React.Fragment>
     );
 
+    const supportedContractCard = (
```

Comment:
Don't we use snake case for variable names? :eyes:

Reply:
yeah, right, will fix it